pkg/exporter/exporter.go

```diff
@@ -277,6 +278,12 @@ func New(server string, timeout time.Duration, logger log.Logger, tlsConfig *tls
 			nil,
 			nil,
 		),
+		directReclaims: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, "", "total_direct_reclaims"),
```

Comment:
Must match Prometheus naming conventions.

```suggestion
			prometheus.BuildFQName(Namespace, "", "direct_reclaims_total"),
```

Reply:
Done